src/app/(user)/dashboard.tsx

```diff
@@ -87,7 +87,7 @@ export default function DashboardEdit(): JSX.Element {
         <View>
             <ScrollView
                 contentInsetAdjustmentBehavior="automatic"
-                style={styles.page}
+                contentContainerStyle={styles.page}
```

Comment:
Dashboard Page does not have enough space to scroll past navbar and see content behind it.

![image](https://github.com/neuland-ingolstadt/neuland.app-native/assets/39560569/a6f5e253-1a3c-4b31-bdf4-d60cc7a6ba56)